priv/templates/phx.gen.auth/forgot_password_live_test.exs

```diff
@@ -12,8 +12,8 @@ defmodule <%= inspect context.web_module %>.<%= inspect Module.concat(schema.web
       {:ok, _lv, html} = live(conn, ~p"<%= schema.route_prefix %>/reset_password")
 
       assert html =~ "Forgot your password?"
-      assert html =~ "Register</a>"
-      assert html =~ "Log in</a>"
+      assert html =~ "/users/register"
```

Comment:
has_element? should work on these and give us the best of both worlds
```suggestion
      assert has_element?(lv, ~s|a[href="#{~p"/users/register"}"]|, "Register")
```

Reply:
resolved in https://github.com/phoenixframework/phoenix/pull/5090/commits/e86b6b372e285f6488a5cdd3f31aa1c88c233cc5